metrics-datadog/src/main/java/org/coursera/metrics/datadog/transport/UdpTransport.java

```diff
@@ -22,6 +24,7 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(UdpTransport.class);
   private final StatsDClient statsd;
+  private final HashMap lastSeenCounters = new HashMap<String,Integer>();
```

Comment:
Hmm... it's in the sub-project POM here:
https://github.com/coursera/metrics-datadog/blob/master/pom.xml#L158 . We
seem to be behind by a major version :/

On Mon, Jan 5, 2015 at 4:25 PM, Cory G Watson <notifications@github.com>
wrote:

> In
> metrics-datadog/src/main/java/org/coursera/metrics/datadog/transport/UdpTransport.java
> <https://github.com/coursera/metrics-datadog/pull/23#discussion-diff-22500646>
> :
>
> > @@ -22,6 +24,7 @@
> >
> >    private static final Logger LOG = LoggerFactory.getLogger(UdpTransport.class);
> >    private final StatsDClient statsd;
> > +  private final HashMap lastSeenCounters = new HashMap<String,Integer>();
>
> re: 1, where is the timgroup statsd client being included? I don't see it
> in the poms and it's arguing with me that it won't work with a long, which
> it clearly does
> <https://github.com/tim-group/java-statsd-client/blob/master/src/main/java/com/timgroup/statsd/StatsDClient.java#L37>
> …
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coursera/metrics-datadog/pull/23/files#r22500646>.
>